packages/argo-admin-react/src/extension-api/data/examples/Data.example.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {extend, render, Text, useData} from '@shopify/argo-admin-react';
+
+function App() {
+  const {productId, variantId} = useData();
+
+  return (
+    <Text>
+      {`
+      Here is my product data:
+      - product id: ${productId}
+      - variant id: ${variantId}
+    `}
+    </Text>
+  );
+}
+
+extend(
+  'Admin::Product::SubscriptionPlan::Add',
+  render(() => <App />),
+);
```

Comment:
It may worth adding the EOF new lines to all these new files so they don't get modified with unrelated code changes later.